### ISSUE TITLE
Can't boot into gmenunx on LF2000

### ISSUE THREAD

**Issue by andymcca** [mac2612/retroleap#40]
If I build LF2000 retroleap with gmenunx included via menuconfig, and then flash the device using sshflash (or sshflash-win), it doesn't boot into gmenunx.  I'm just left at the 'Leapfrog' white logo screen.

If I disable the spawning entirely (remove RA and gmenunx sections from respawn_menu.sh), and then try running gmenunx via a connected terminal, I get output to the terminal as if gmenunx is loading up successfully but the screen doesn't change.

I suspect it's possibly due to screen display mode / need to change the framebuffer?

**Comment by andymcca** [comment]
Update - have got something on the display now, but gmenunx appears to be changing the screen resolution to 512x384 on loading.  

I am setting the active framebuffer to fb0 prior to loading, which is 320x240x16bpp.  Am also blanking fb1 and fb2.

**Comment by andymcca** [comment]
Fixed it with a hacky workaround.  LF2000 doesn't currently have a platform specification in the gmenunx code in mac's fork (LF1000 already specified in platform.cpp / platform.h / lf1000.h).  For my own purposes I just forced the platform detection to always be LF1000 to get it working as it's similar enough.  This did the trick and resolved the display issue.  

I also blanked all Framebuffers except fb0 in the Retroleap respawn_menu.sh script to ensure the 16 bit colour framebuffer is used.  Failing to do this (or just partially doing it) leave the Leapfrog logo visible whenever gmenunx goes into sleep mode.
